### DISCUSSION TITLE
Gossip non-transactions

### DISCUSSION THREAD

**Discussion by aphexmunky** [ava-labs/hypersdk#357] [Q&A]
Is there a way to gossip across validator nodes content that I don't want persisting on any level? Like a non-persistent transaction to build up some transient state that doesn't need to be perfect. 

So for example if i wanted to build a trollbox that any node could serve. I don't care about the messages enough to persist anything. I'll take responsibility for DoS management. It doesn't matter to me that every node serves the right messages in any particular order. I'm not trying to build a trollbox but just as a use-case

**Comment by patrick-ogrady** [comment]
> Is there a way to gossip across validator nodes content that I don't want persisting on any level? Like a non-persistent transaction to build up some transient state that doesn't need to be perfect.

This is not exposed in the `Controller` interface yet but is very possible to do. Would you mind creating an issue for this feature request?

We implemented a network handler that multiplexes traffic across multiple "callbacks" (https://github.com/ava-labs/hypersdk/blob/main/network/network.go) but only use it for tx gossip, state sync, warp messaging right now.
